test/modules/TwinTester/TwinAllOperationsInitializer.cs

```diff
@@ -61,7 +61,14 @@ public static async Task<TwinAllOperationsInitializer> CreateAsync(RegistryManag
                 else
                 {
                     Logger.LogInformation("Existing storage detected. Initializing reported / desired property update counters.");
-                    initializedState = new TwinState { ReportedPropertyUpdateCounter = GetNewPropertyCounter(reportedPropertyUpdates), DesiredPropertyUpdateCounter = GetNewPropertyCounter(desiredPropertyUpdates), TwinETag = twin.ETag, LastTimeOffline = DateTime.MinValue };
+                    initializedState = new TwinState
+                    {
+                        ReportedPropertyUpdateCounter = GetNewPropertyCounter(reportedPropertyUpdates),
+                        DesiredPropertyUpdateCounter = GetNewPropertyCounter(desiredPropertyUpdates),
+                        TwinETag = twin.ETag,
+                        LastTimeOffline = DateTime.MinValue,
+                        LastTimeOfEdgeRestart = DateTime.MinValue
+                    };
```

Comment:
I noticed that for other twin initializes we do not initialize most of these fields. I feel like we might want to change this. Let me know if you want me to make the change.

Reply:
What is the purpose of these fields?

Reply:
Counters: Update with increasing sequence numbers
TwinEtag: needed for updating the twin
DateTimes: Failure tolerance

Reply:
For counters and datetimes, they will set to defaults, which should be 0 and datetime.minvalue; which is fine.  But I don't know ETag, it will set to null and so ETag is not required to update for twin method calls? What is the purpose of ETag?

Reply:
Based on this, it will work if we leave every field but the ETag uninitialized.

The ETag needs to be initialized in order for any desired property update to occur. Other twin operation initializers initialize ETag, but not the other fields.